src/include/distributed/multi_physical_planner.h

```diff
@@ -326,6 +326,7 @@ extern bool TaskListMember(const List *taskList, const Task *task);
 extern List * TaskListDifference(const List *list1, const List *list2);
 extern List * AssignAnchorShardTaskList(List *taskList);
 extern List * FirstReplicaAssignTaskList(List *taskList);
+extern bool CoPartitionedTables(Oid firstRelationId, Oid secondRelationId);
```

Comment:
I think this shouldn't be the last function considering the definition/declaration orders